### ISSUE TITLE
UI improvement to concluded license handling

### ISSUE THREAD

**Issue by yaobinshi** [fossology/fossology#373]
---

Author Name: **Raino Lintulampi** 
Original Redmine Issue: 8103, http://www.fossology.org/issues/8103
Original Date: 2014/12/31

---

I propose following changes for handling concluded license in
different views.
1. Adding new user permission level ("Audit"). The user has to 
   have this permission level in order to change/review licenses.
2. Removing the 'Edit concluded license' button from View License
   view if concluded license has already been set. In that case
   additional change has to be done via Audit menu.
3. Adding possibility to set concluded license directly from
   List Files for License in case concluded license has not been 
   set. Also it could be useful to set concluded license for 
   multiple files at the same time.
4. Concluded license is shown in addition to scanner result in 
   List Files for License view.
5. Restoring the capability to change nomos scanner result.

All these changes except setting concluded license for multiple
files are demonstrated in the enclosed patch.

**Comment by pguttmann** [comment]
Attachment - http://www.fossology.org/attachments/4293/fossology-2.6.2_patch.tar.gz

**Comment by mcjaeger** [comment]
I am sorry, this ticket was left out during reworking of the ticket landscape.

1. New 'Audit' role is like an Admin without the Admin part but just covering the licenses part? Sounds OK.
2. Removing this button: I am not convinced because that would rise the question how to correct set licenses, also, the I guess that it for the "old 2.6 UI"?
3. Is done in the browse view using the edit functionality, shall this be possible on the licenses / affect files in the left histogram view?
4. Fully agree that mentioned view is not useful to this regards, however, the main license table (right part of the license browser main view does that already. Question is how to clean up the old view (maybe remove it from code). Please see the screenshot, how it looks like today:

![2015-11-19 10_04_34-139 25 105 20 - remote desktop connection](https://cloud.githubusercontent.com/assets/3418977/11266516/53264378-8ea5-11e5-865b-a1286747d3ff.png)

5. That does not appear useful to me, because the scanner results should be fix and the user has the possibility to conclude another license.

So, in summary now:

1. New Audit role for working on lienses only (so there is a full admin and a license admin)
2. Adding functionality to apply a direct "Edit" not only for folders, but also for file with the same license findings.

?

**Comment by maximilianhuber** [comment]
duplicate of /related to #39

**Comment by shaheemazmalmmd** [comment]
With @mcjaeger's comment everything is clarified here. closing the issue.
